js/back.js

```diff
@@ -65,6 +65,38 @@
 
         return accumulator;
       }, {});
+      if (browsingDataObj.cookies) {
+        // delete cookies here + ignore whitelisted cookies
+        getStorage("cookieWhitelist", function(data) {
+          var domain_list = data.cookieWhitelist
+          getAllCookies({}, function(cookies)
+          {
+            var callbackCount = 0;
+            for (var i = 0; i < cookies.length; i++)
```

Comment:
Suggestion: if we don't use index, we can use `for..of` loop instead, in most of the cases `for` loop were used as initial code was written even before `for..of` were introduced. This way the code will be more readable.

```
for (const cookie of cookies)
{
  ...
}
```